frontend/public/locales/en/common.json

```diff
@@ -101,6 +101,26 @@
   "apps-by-projectgroup": "Apps in the {{projectgroup}} group",
   "search-for-query": "Search for '{{query}}'",
   "apps-description": "Explore the Linux apps available on Flathub",
+  "applications-description": "An app store and build service for Linux",
+  "moderation-dashboard": "Moderation Dashboard",
```

Comment:
I'm not convinced moderation should be translated. Would like to hear what others think.

Reply:
Oh I guess this new panel is not just for moderators, but for app authors too. NVM then...

Reply:
App authors won't be able to see all of these strings. I can leave the ones in the moderator-only pages untranslated.

Reply:
Please do.